src/screens/Menu/Menu.js

```diff
@@ -0,0 +1,298 @@
+  SECURITY_SETTINGS,
+  RECOVERY_SETTINGS,
+  APP_SETTINGS,
+  COMMUNITY_SETTINGS,
+  ADD_EDIT_USER,
+} from 'constants/navigationConstants';
+import { LIGHT_THEME } from 'constants/appSettingsConstants';
+import { logoutAction } from 'actions/authActions';
+
+import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
+import type { Theme } from 'models/Theme';
+import type { NavigationScreenProp } from 'react-navigation';
+import type { BackupStatus } from 'reducers/walletReducer';
+
+type Props = {
+  navigation: NavigationScreenProp<*>,
+  theme: Theme,
+  user: Object,
+  backupStatus: BackupStatus,
+  logoutUser: () => void,
+};
+
+type State = {
+  visibleModal: ?string,
+};
+
+const headerLogo = require('assets/images/landing-pillar-logo.png');
+const headerLogoDarkMode = require('assets/images/landing-pillar-logo-dark-theme.png');
+
+const Footer = styled.View``;
+
+const LinksSection = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: ${spacing.mediumLarge}px 0;
+`;
+
+const LogoutSection = styled.View`
+  border-top-color: ${themedColors.tertiary};
+  border-top-width: 1px;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: ${spacing.large}px;
+`;
+
+const HeaderLogo = styled(CachedImage)`
+  width: 40px;
+  height: 20px;
+`;
+
+const LogoutIcon = styled(Icon)`
+  color: ${themedColors.negative};
+  ${fontStyles.regular};
+  margin-right: 5px;
+`;
+
+const LegalTextLink = styled(TextLink)`
+  ${fontStyles.regular};
+`;
+
+const LogoutTextLink = styled(TextLink)`
+  color: ${themedColors.negative};
+  ${fontStyles.regular};
+`;
+
+const StyledEmoji = styled(Emoji)`
+  margin-right: 10px;
+`;
+
+
+class Menu extends React.Component<Props, State> {
+  state = {
+    visibleModal: null,
+  }
+
+  getMenuItems = () => {
+    const {
+      theme, navigation, backupStatus,
+    } = this.props;
+    const colors = getThemeColors(theme);
+    const isBackedUp = backupStatus.isImported || backupStatus.isBackedUp;
+
+
+    return [
+      {
+        key: 'securitySettings',
+        title: 'Security settings',
+        emoji: 'rotating_light',
+        card: true,
+        action: () => navigation.navigate(SECURITY_SETTINGS),
+      },
+      {
+        key: 'recoverySettings',
+        title: 'Recovery settings',
+        labelBadge: !isBackedUp && {
+          label: 'Warning',
+          color: colors.negative,
+        },
+        emoji: 'mage',
+        card: true,
+        action: () => navigation.navigate(RECOVERY_SETTINGS),
+      },
+      {
+        key: 'userProfile',
+        title: 'User profile',
+        emoji: 'male-singer',
+        card: true,
+        action: () => navigation.navigate(ADD_EDIT_USER),
+      },
+      {
+        key: 'appSettings',
+        title: 'App settings',
+        emoji: 'gear',
+        card: true,
+        action: () => navigation.navigate(APP_SETTINGS),
+      },
+      {
+        key: 'referFriends',
```

Comment:
This was supposed to be for devs only, as far as I remember. Referral program is currently in works :)

Reply:
Can we add comment here for this?